pwa/app/routes/team.$teamNumber.tsx

```diff
@@ -366,21 +363,211 @@ export default function TeamPage(): JSX.Element {
+            <Stat
+              label={pluralize(awards.length, 'Award', 'Awards', false)}
+              value={awards.length}
+            />
+          )}
+
+          {highScoreMatch && (
+            <TooltippedStat
+              label="High Score"
+              value={highScoreMatch.score}
+              tooltip={`${highScoreMatch.match.key} - ${highScoreMatch.alliance.team_keys.map((k) => k.substring(3)).join('-')}`}
+            />
+          )}
+        </div>
+
+        <Table className="min-w-[300px] table-fixed [&_tr]:border-b-0">
+          <TableHeader>
+            <TableRow>
+              <TableHead className="text-center"></TableHead>
+              <TableHead className="text-center">Quals</TableHead>
+              <TableHead className="text-center">Playoffs</TableHead>
+              <TableHead className="text-center">Overall</TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            <TableRow>
+              <TableHead>Official</TableHead>
+              <RecordCell record={officialRecords.quals} />
+              <RecordCell record={officialRecords.playoff} />
+              <RecordCell record={officialRecord} />
+            </TableRow>
+
+            <TableRow>
+              <TableHead>Unofficial</TableHead>
+              <RecordCell record={unofficialRecords.quals} />
+              <RecordCell record={unofficialRecords.playoff} />
+              <RecordCell record={unofficialRecord} />
+            </TableRow>
+
+            <TableRow>
+              <TableHead>Combined</TableHead>
+              <RecordCell record={combinedQuals} />
+              <RecordCell record={combinedPlayoff} />
+              <RecordCell record={combinedRecord} />
+            </TableRow>
+          </TableBody>
+        </Table>
+      </div>
+    </div>
+  );
+}
+
+function RecordCell({ record }: { record: WltRecord }) {
+  return (
+    <TableCell className="text-center">
+      <TooltipProvider>
+        <Tooltip>
+          <TooltipTrigger asChild>
+            <div className="cursor-pointer">{stringifyRecord(record)}</div>
+          </TooltipTrigger>
+          <TooltipContent side="top">
+            {(winrateFromRecord(record) * 100).toFixed(0)}% winrate
+          </TooltipContent>
+        </Tooltip>
+      </TooltipProvider>
+    </TableCell>
+  );
 }
 
 function Stat({ label, value }: { label: string; value: string | number }) {
   return (
-    <div className="mx-auto flex w-[calc(50%-0.5rem)] min-w-[14ch] flex-col text-center sm:w-[calc(20%-0.8rem)]">
+    <div className="mx-auto flex min-w-[16ch] flex-col text-center">
       <dt className="text-gray-500">{label}</dt>
-      <dd className="order-first text-3xl font-semibold tracking-tight text-gray-900">
+      <dd className="order-first text-2xl font-semibold tracking-tight text-gray-900">
         {value}
       </dd>
     </div>
   );
 }
+
+// For some reason you can't use the <Stat /> component inside the Trigger with asChild
```

Comment:
Is `asChild` necessary? I just tried wrapping `<Stat />` without `asChild` and it seems to work?

Reply:
No idea what I did wrong originally, but yeah seems to work when I try again